src/pages/EmergencyViewPage/EmergencyViewPage.tsx

```diff
@@ -121,16 +121,17 @@ export const EmergencyViewPage: React.FC = () => {
   }
 
   return (
-    <WithNavigationBar withSchedule={false}>
-      <div
-        className="EmergencyView__banner"
-        style={{
-          backgroundImage: `url(${validBannerImageUrl})`,
-        }}
-      ></div>
+    <WithNavigationBar withSchedule={false} hasBackButton={false}>
       <div className="EmergencyView">
         <EmergencyViewTabs updateTab={updateTab} selectedTab={selectedTab} />
-        <div className="EmergencyView_main">
+        <div
+          className="EmergencyView_main"
```

Comment:
```suggestion
          className="EmergencyView__main"
```

And the appropriate SCSS file selector

Reply:
fixed 308745f